packages/hash/frontend/src/blocks/page/createFormatPlugins/LinkModal.tsx

```diff
@@ -69,7 +69,16 @@ export const LinkModal: FunctionComponent<LinkModalProps> = ({
               >
                 <LanguageIcon className={tw`!text-base mr-1`} />
                 <div>
-                  <p className={tw`text-sm leading-none mb-0.5`}>
+                  <p
+                    className={tw`text-sm leading-none break-all mb-0.5`}
+                    style={{
+                      display: "-webkit-box",
+                      WebkitLineClamp: 4,
+                      WebkitBoxOrient: "vertical",
```

Comment:
Does this work in firefox etc?

Reply:
Yup, doesn't work with Internet Explorer tho.
https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-line-clamp#browser_compatibility
https://developer.mozilla.org/en-US/docs/Web/CSS/box-orient#browser_compatibility